### PR TITLE
Add extra padding to finances and marketing windows

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Like with #6502, this PR adds some extra padding to the financial accounts and marketing tabs of the finances window. Effectively, this makes each row 12px instead of 10px.

My primary motivation for changing these was to make the Japanese translation look a little bit better, but I believe it looks much better for English, too, as the information is less cramped this way.

If this PR is received well, I plan to tackle other windows as well — e.g. the guests and staff windows.

**Financial accounts:**
English: [before](https://user-images.githubusercontent.com/604665/31677516-4f70a8ee-b36b-11e7-8b4e-d8a3019afed5.png), [after](https://user-images.githubusercontent.com/604665/31677514-4f4a2a52-b36b-11e7-823c-5c550788bbf0.png)
Japanese: [before](https://user-images.githubusercontent.com/604665/31677519-4fc6c6b6-b36b-11e7-8356-78847711e89f.png), [after](https://user-images.githubusercontent.com/604665/31677517-4f932838-b36b-11e7-826b-7baf98104549.png) 

**Marketing:**
English: [before](https://user-images.githubusercontent.com/604665/31677713-f98625ac-b36b-11e7-8c08-6e8a0a2ba703.png), [after](https://user-images.githubusercontent.com/604665/31677711-f969062a-b36b-11e7-9bd2-a145c1775a08.png)
Japanese: [before](https://user-images.githubusercontent.com/604665/31677716-f9be5a12-b36b-11e7-9c30-f463d4518567.png), [after](https://user-images.githubusercontent.com/604665/31677714-f99e7abc-b36b-11e7-9f1e-90afd630aa47.png)

### CODE REVIEW

File: src/openrct2/interface/window.h [hunk 1]
```diff
@@ -35,6 +35,10 @@ struct TitleSequence;
 struct TextInputSession;
 typedef struct scenario_index_entry scenario_index_entry;
 
+#define ROW_HEIGHT 12
```
Comment (Gymnasiast) [inline src/openrct2/interface/window.h]: Perhaps rename this this SCROLLABLE_ROW_HEIGHT?

Comment (AaronVanGeffen) [reply src/openrct2/interface/window.h]: Could do, but then I'll need another one for the non-scrollable ones — like in the list of active marketing campaigns. What do you think?

Comment (Gymnasiast) [reply src/openrct2/interface/window.h]: Sounds like a good idea. After all, those are different widgets.

Comment (AaronVanGeffen) [reply src/openrct2/interface/window.h]: Done.

**Pull request merged**